src/server.js

```diff
@@ -13,6 +14,11 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 app.use(bodyParser.raw());
 
+app.use('*' , cors());
```

Comment:
How is this line different from the `app.options('*', cors())` line? Is it possible to get rid of one these lines?